Quark_Sketch/src/arduino/libraries/SPI_LIB.h

```diff
@@ -1,15 +1,17 @@
 /*
```

Comment:
Just to confirm, you have re-written this file from scratch without referencing the old LGPL version right?